src/main/webapp/js/user-page-loader.js

```diff
@@ -97,8 +97,9 @@ function buildMessageDiv(message) {
   const headerDiv = document.createElement('div');
   headerDiv.classList.add('message-header');
   headerDiv.appendChild(document.createTextNode(
-      message.user + ' - ' + new Date(message.timestamp)));
-
+      message.user + ' - ' +
```

Comment:
suggestion: since this expression is quite long now, it could be a bit hard to understand when going through the code quickly. How about initialize the message first and then use it here, e.g.:

```
String message = message.user + ' - ' + new Date(message.timestamp) +
      ' Sentiment Score: [' + message.sentimentScore + ']';
headerDiv.appendChild(document.createTextNode(message));
```